infra/build/functions/base_images.py

```diff
@@ -65,14 +67,60 @@ def _get_base_image_steps(images, tag_prefix=TAG_PREFIX):
   return steps
 
 
+def _get_introspector_base_images_steps(images, tag_prefix=TAG_PREFIX):
+  """Returns build steps for given images version of introspector"""
+  steps = [{
+      'args': [
+          'clone',
+          'https://github.com/ossf/fuzz-introspector.git',
+      ],
+      'name': 'gcr.io/cloud-builders/git',
+  }]
+
+  steps.append({
+      'name':
+          'gcr.io/oss-fuzz-base/base-runner',
+      'args': [
+          'bash', '-c',
+          ('cd fuzz-introspector/ && cd oss_fuzz_integration/'
+           ' && sed -i \'s/\.\/infra\/base\-images\/all.sh/'
+           '#\.\/infra\/base\-images\/all.sh/\''
```

Comment:
Let's upstream as many of the image diffs as possible to avoid this hacky and difficult to maintain patching. Could you create another PR to incorporate the changes in https://github.com/ossf/fuzz-introspector/blob/main/oss_fuzz_integration/oss-fuzz-patches.diff ? 

We can guard certain things with env vars if necessary.